### PR TITLE
Add tutorial of adding badges to tabbar icons

### PR DESCRIPTION
**Pull request by sunnylqm**
Resolves https://github.com/react-navigation/react-navigation.github.io/issues/81
Snack preview: https://snack.expo.io/@react-navigation/tabs-with-icons-v3

### CODE REVIEW

File: website/versioned_docs/version-3.x/tab-based-navigation.md [hunk 1]
```diff
@@ -85,14 +88,61 @@ export default createBottomTabNavigator(
...
+```js
+export default class IconWithBadge extends React.Component {
+  render() {
+    const { name, badgeCount, color, size } = this.props;
+    return (
+      <View style={{ width: 24, height: 24, margin: 5 }}>
+        <Ionicons name={name} size={size} color={color} />
+        { badgeCount > 0 && (
+          <View style={{
+            // Absolute positioning ref: https://developer.mozilla.org/en-US/docs/Web/CSS/position#Absolute_positioning
```
Comment (brentvatne) [inline website/versioned_docs/version-3.x/tab-based-navigation.md]: can we condense this comment to one line? maybe just `// If you're using react-native < 0.57 overflow outside of parent will not work on Android, see https://git.io/fhLJ8`

File: docs/tab-based-navigation.md [hunk 2]
```diff
@@ -84,14 +87,61 @@ export default createBottomTabNavigator(
 );
 ```
 
-<a href="https://snack.expo.io/@react-navigation/tabs-with-icons-v3" target="blank" class="run-code-button">&rarr; Run this code</a>
+<a href="https://snack.expo.io/@sunnylqm/tabs-with-icons-v3" target="blank" class="run-code-button">&rarr; Run this code</a>
```
Comment (brentvatne) [inline docs/tab-based-navigation.md]: I updated the code here to clean it up a little bit: https://snack.expo.io/@react-navigation/tabs-with-icons-v3

can you update the copy of it in these docs to match?

File: website/versioned_docs/version-3.x/tab-based-navigation.md (hunk 1 above)
Comment (brentvatne) [reply website/versioned_docs/version-3.x/tab-based-navigation.md]: I updated the comment in https://snack.expo.io/@react-navigation/tabs-with-icons-v3 let me know if that works for you

Comment (sunnylqm) [reply website/versioned_docs/version-3.x/tab-based-navigation.md]: Sure. Updated.

**Comment by brentvatne** [comment]
thanks, this should be helpful for people :)

**Pull request merged**